src/middlewared/middlewared/plugins/disk.py

```diff
@@ -134,6 +138,13 @@ async def disk_extend(self, disk, context):
         else:
             disk.pop('passwd')
             disk.pop('kmip_uid')
+
+        disk['supports_smart'] = None
+        if context['supports_smart']:
+            disk['supports_smart'] = disk['name'].startswith('nvme') or bool(RE_SMART_AVAILABLE.search(
```

Comment:
Please make this `True` always on enterprise hardware like we do in the `SMARTCTX` namedtuple.